spec/std/process_spec.cr

```diff
@@ -228,6 +228,20 @@ describe Process do
     process.terminated?.should be_true
   end
 
+  it "checks for existence after terminate" do
+    process = Process.new("yes")
+    process.exists?.should be_true
+    process.terminated?.should be_false
+
+    process.terminate
+    process.exists?.should be_true
+    process.terminated?.should be_false
```

Comment:
This seems like a race condition to me. If the program gets rescheduled between the `terminate` call and the `exists?` check then this spec will fail. I suspect that running this test in a loop on a loaded machine will already fail.

I'm not sure there's an appropriate way to test this.

Reply:
Was write access given to the branch? Could someone simply remove this spec and merge the PR?